ports/libsimpleservo/capi/src/lib.rs

```diff
@@ -854,4 +855,14 @@ impl HostTrait for HostCallbacks {
         let contents_str = c_str.to_str().expect("Can't create str");
         Some(contents_str.to_owned())
     }
+
+    fn on_devtools_started(&self, port: Result<u16, ()>) {
+        match port {
+            Ok(p) => {
+                debug!("Devtools Server running at {}", p);
+                (self.0.on_devtools_started)(p.into())
+            },
+            Err(()) => eprintln!("Error running devtools server"),
```

Comment:
Error can't be lost here, we need to let the embedder know that something didn't work. Maybe the callback `on_devtools_started` can take 2 arguments. A result, and a port.

Look at `CPromptResult` for example.

```rust
  #[repr(C)]
  pub enum CDevtoolsServerState {
    Started,
    Error,
  }
```